### PR TITLE
Running Engagements

### PR DESCRIPTION
**Pull request by GaryPWhite**
Closes... a lot.
#10 
#11 
#12 
#13 
#14 
#15 
#16 
#17 

Fills out our documentation for running engagements.

### CODE REVIEW

File: src/docs/docs.css [hunk 1]
```diff
@@ -40,7 +40,11 @@
   line-height: 32pt;
   color: #612578;
 }
-
+.doc-container h2 > a {
+  font-size: 32pt;
+  line-height: 32pt;
+  color: #612578;
+}
```
Comment (GaryPWhite) [inline src/docs/docs.css:47]: links look better in these headers as the same color trust me

File: src/components/header.js [hunk 2]
```diff
@@ -109,20 +109,22 @@ const Header = () => (
...
+        <Col sm lg="3" style={{
+          marginTop: "3px"
+        }}>
           <Link to="/">
             <StaticImage
-              src="../images/aux.png"
-              height={40}
+              src="../images/auxeng.png"
+              height={35}
               formats={["AUTO"]}
```
Comment (GaryPWhite) [inline src/components/header.js:128]: formatting for the new aux eng banner image

File: src/docs/theory/marketing.md [hunk 3]
```diff
@@ -15,3 +15,16 @@ featured: ../images/featured/theory.png
...
+- Figure out early what patterns we want to change
+- Start changing behaviors
+- Make any big changes early
+  - Prioritizing reviews
+  - Investing in pipelines / tests
+  - Code formatting / linting changes
+  - Now is the time to disrupt
+- Reinforce habits
+  - Spend 2+ months reinforcing habits / technology usage
+  - Try not to introduce large behavioral changes towards middle/end
```
Comment (GaryPWhite) [inline src/docs/theory/marketing.md:30]: moved some of this from running into theory

File: src/docs/running/sourcing.md [hunk 4]
```diff
@@ -3,22 +3,109 @@ title: "Sourcing"
...
+
+We regularly have teams approach us; looking for advice on building new
+services. We will explain to the team what our AuxEng program is and give
+them some general direction on how to proceed with their project.
+
+Most questions we get and most outline we give stem from the mission of the
+AuxEng team that sponsors the engineer. For example, we might have a team that
+heavily focuses on best practices for Python. We outline how we make life
+better for engineers that use our recommended tools, how we help setup automation
+and quality of life improvements for folks we engage with, and [generally pitch](./expectations.md#prepare-the-pitch)
```
Comment (lelia) [inline src/docs/running/sourcing.md:23]: Broken link (I was able to get http://localhost:8000/running/expectations/#prepare-the-pitch to work locally, though)

File: src/docs/running.md [hunk 5]
```diff
@@ -3,153 +3,83 @@ title: "Running Engagements"
...
-been fully communicated to the entire team, so we make sure to reiterate it
-here.
+- **[Weekly Retros](./executing#weekly-retros)**
 
-Finally we talk through the project timeline, and communicate expectations about
-when the project MVP will be delivered.
+_How is/are the engagement, and the goals within, moving along?_
 
-## Weekly Retros
+- **[Project Retros / NPS](executing/project-retros)**
```
Comment (lelia) [inline src/docs/running.md:63]: also borken - works with http://localhost:8000/running/executing#project-retros

File: src/docs/running/wrapping-up.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
+---
```
Comment (lelia) [inline src/docs/running/wrapping-up.md:1]: For some reason, a lot of the pages/sub-headers on this page are loading extremely slowly on my local instance, and gets stuck on `Preparing requested page` for quite some time.  It doesn't seem to happen with any of the other pages, but I'm not sure what the bottleneck could be. I'll see if i can reliably reproduce it.

Comment (lelia) [reply src/docs/running/wrapping-up.md:1]: Interestingly, some folks have reported this being a unique issue with M1 macbooks - wonder if that could be the culprit? https://stackoverflow.com/questions/66681392/gatsby-error-when-creating-a-new-route-preparing-requested-page

File: src/docs/running/expectations.md [hunk 7]
```diff
@@ -3,10 +3,69 @@ title: "Engagement Expectations"
...
+
+## Set a Baseline
+
+Setting expectations about our team and what will and won't happen in an aux
+engagement is important. We think through these ideas carefully and publish our
+opinion of how they apply to our team in a conspicuous place. Usually on an
+internal webpage about our team and practices; included in any e-mails or
+internal messages about our team to prospective "away" teams.
+
+> **Aux engineer will not do the hard work**
```
Comment (lelia) [inline src/docs/running/expectations.md:24]:
```suggestion
> **Aux Engineer will not do the hard work**
```

File: src/docs/running/expectations.md [hunk 8]
```diff
@@ -3,10 +3,69 @@ title: "Engagement Expectations"
...
+should come second to achieving some particular engagement goal. AuxEng is used
+at Wayfair to proliferate usage of a tool or practice, not to burn down backlogs
+ for any team.
+> **Weekly retros and schedules are important. Critical feedback is strongly encouraged.**
+It's important to understand what scheduled rituals are important for you and
+your team -- and encourage that replacements or alternatives have to achieve the
+same goals. For example, retros are a time to understand progress and get
+feedback. Without a replacement, everyone should attend and engage in retros,
+or the model does not mutually benefit engagement "host" and "away" teams.
+> **If failure modes persist, the engagement will ended early**
```
Comment (lelia) [inline src/docs/running/expectations.md:41]:
```suggestion
> **If failure modes persist, the engagement will ended early.**
```

File: src/docs/running/expectations.md [hunk 9]
```diff
@@ -3,10 +3,69 @@ title: "Engagement Expectations"
...
+teams for extended periods of time, and we have to have engagements that are
+worth doing for our team's priority.
 
-- Aux engineer will not do the hard work
-- Diffusion is the primary goal and project success is the secondary goal.
-  - Sacrificing diffusion for the sake of reaching the project goals is not viable
-- Weekly retros are important, and critical feedback is strongly encouraged.
-- If failure modes persist, the engagement will ended early
+This document will outline how we approach engagement in an organization, and
+high-level points to reiterate in a document that should come with any [advisor](./roles.md)
```
Comment (lelia) [inline src/docs/running/expectations.md:13]: Broken link

```suggestion
high-level points to reiterate in a document that should come with any [advisor](../roles)
```

File: src/docs/running/sourcing.md [hunk 10]
```diff
@@ -3,22 +3,109 @@ title: "Sourcing"
...
-- Selection
-  - Score on 3 criteria
-    - Business Impact
-    - Technical Impact
-    - Team fit
-  - AuxEng team individually scores proposal 0-10 on each criteria
-  - Discuss "Why" for each rating
-  - Participants get an opportunity to shift their votes
-  - Use your judgment, or be strictly mathematical. Either way you have the scoring data to measure
+After drumming up interest and [clarifying a team's mission](./expectations.md),
```
Comment (lelia) [inline src/docs/running/sourcing.md:6]: Broken link

```suggestion
After drumming up interest and [clarifying a team's mission](../expectations),
```

File: src/docs/running/sourcing.md (hunk 4 above)
Comment (lelia) [inline src/docs/running/sourcing.md:23]: Broken link

```suggestion
and quality of life improvements for folks we engage with, and [generally pitch](../expectations#prepare-the-pitch)
```

File: src/docs/running.md [hunk 11]
```diff
@@ -3,153 +3,83 @@ title: "Running Engagements"
...
-to look for projects that we feel will have a measurable impact on Wayfair's
-performance. We often look for projects that can associate a dollar amount with
-getting their minimum viable product into production.
+- **[One-Pager](./sourcing#one-pager)**
 
-A major refactor of a legacy system is an example of a project that we would
-consider to have low business value. It might solve a lot of problems for
-engineers, but it most likely has little to no immediate value for the business
-itself.
+One-Pagers describe an team's problem, and proposed solution / practices
```
Comment (lelia) [inline src/docs/running.md:27]:
```suggestion
One-Pagers describe a team's problem, and proposed solution / practices
```

File: src/docs/running.md (hunk 5 above)
Comment (lelia) [inline src/docs/running.md:63]: Broken link

```suggestion
- **[Project Retros / NPS](./executing#project-retros)**
```

File: src/docs/running/sourcing.md [hunk 12]
```diff
@@ -3,22 +3,109 @@ title: "Sourcing"
...
+
+After receiving a one-pager, We meet internally to rate the one-pager on
+three metrics; business value, technical value, and team fit.
+
+## Checking Project Fit
+
+After some introductions and receiving a project brief via a one-pager, we
+collect our team and make decisions on if we should proceed. To avoid bias
+for or against a particular engagement, we will judge based on three tenets
+important to the success of any project. Business Value, Technical Value, and
```
Comment (lelia) [inline src/docs/running/sourcing.md:56]:
```suggestion
important to the success of any project: Business Value, Technical Value, and
```

File: src/docs/running/planning.md [hunk 13]
```diff
@@ -3,14 +3,78 @@ title: "Planning"
...
-    - Metric that matters the most
-    - Business impact
-    - Standard OKRs
-  - Technical goals
-    - Test coverage %
-    - Setup documentation pipeline
-    - etc.
-- AuxEng builds some domain familiarity
-  - Dedicated Time to learn and familiarize
+By the planning phase, we have [set expectations](./expectations.md) for others
```
Comment (lelia) [inline src/docs/running/planning.md:6]: Broken link

```suggestion
By the planning phase, we have [set expectations](../expectations) for others
```

File: src/docs/running/planning.md [hunk 14]
```diff
@@ -3,14 +3,78 @@ title: "Planning"
...
+to interact with our team, and [successfully sourced](./sourcing.md) a  project
```
Comment (lelia) [inline src/docs/running/planning.md:7]: Broken links

```suggestion
By the planning phase, we have [set expectations](../expectations) for others
to interact with our team, and [successfully sourced](../sourcing) a  project
```

File: src/docs/running/executing.md [hunk 15]
```diff
@@ -0,0 +1,107 @@
+---
+title: "Executing"
+featured: ../images/featured/running.png
+---
+
+We've done it! We've marketed ourselves and set [expectations](./expectations).
+The hard work of [sourcing](./sourcing) and [planning](./planning) is finished,
```
Comment (lelia) [inline src/docs/running/executing.md:7]: Broken links

```suggestion
We've done it! We've marketed ourselves and set [expectations](../expectations).
The hard work of [sourcing](../sourcing) and [planning](../planning) is finished,
```

File: src/docs/running/executing.md [hunk 16]
```diff
@@ -0,0 +1,107 @@
...
+our team is now at the meat and potatoes: doing the aux engagement.
+
+## Weekly Schedule
+
+The first part to get straight with a host team is the schedule we'll work
+together.
+
+### 80% Time
+
+We recommend using "80%" time. An [Aux Engineer](./roles) should be with the
```
Comment (lelia) [inline src/docs/running/executing.md:17]: Broken link

```suggestion
We recommend using "80%" time. An [Aux Engineer](../roles) should be with the
```

File: src/docs/running/wrapping-up.md [hunk 17]
```diff
@@ -0,0 +1,82 @@
...
+us and reconsider the ones that didn't work. Everything we do is up for debate,
+and we use it all to achieve the mission and goals of AuxEng and our team.
+
+It should be mentioned that this process can take a toll on [engineers and advisors](./roles).
+Your Milage Will Vary, but we generally don't expect engineers to want to do
+more than 2 engagements back to back. Anything more will leave folks feeling
+disconnected from the team running Aux engagements.
+
+When you're ready with another good idea, fresh engineers, and a strong fit,
+[roll it back](../running)!
```
Comment (lelia) [inline src/docs/running/wrapping-up.md:82]: Broken links

```suggestion
It should be mentioned that this process can take a toll on [engineers and advisors](../roles).
Your Milage Will Vary, but we generally don't expect engineers to want to do
more than 2 engagements back to back. Anything more will leave folks feeling
disconnected from the team running Aux engagements.

When you're ready with another good idea, fresh engineers, and a strong fit,
[roll it back](../)!
```

**Review by lelia — APPROVED** [review]

**Pull request merged**